### DISCUSSION TITLE
Since THE WEIGHT CLASS I DOWNLOADED IS 80 and I need TO BE 2, how do I not load this layer for training, or what is a good idea

### DISCUSSION THREAD

**Discussion by weikai520** [open-mmlab/mmdetection#9109] [Q&A]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 🐞 Describe the bug

Since THE WEIGHT CLASS I DOWNLOADED IS 80 and I need TO BE 2, how do I not load this layer for training, or what is a good idea

raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for YOLOX:
size mismatch for bbox_head.multi_level_conv_cls.0.weight: copying a param with shape torch.Size([80, 128, 1, 1]) from checkpoint, the shape in current model is torch.Size([2, 128, 1, 1]).

### Environment

size mismatch for bbox_head.multi_level_conv_cls.0.weight: copying a param with shape torch.Size([80, 128, 1, 1]) from checkpoint, the shape in current model is torch.Size([2, 128, 1, 1]).

### Additional information

size mismatch for bbox_head.multi_level_conv_cls.0.weight: copying a param with shape torch.Size([80, 128, 1, 1]) from checkpoint, the shape in current model is torch.Size([2, 128, 1, 1]).

**Comment by ZwwWayne** [comment]
In training, simply setting `load_from` to be the path of the weight should be fine, those mismatched weights will be ignored.

**Comment by weikai520** [comment]
It's still the same mistake Sorry, trying doesn't work
